tests/EntityTests.cpp

```diff
@@ -6,6 +6,7 @@
 #include "Component/Position/PositionComponent.hpp"
 #include <catch2/catch.hpp>
 
+using namespace BBM;
```

Comment:
You don't need to use the same namespace twice.

Reply:
It was a merge isue